shared/actions/teams.js

```diff
@@ -388,11 +384,7 @@ const _createNewTeamFromConversation = function*(
           })
         }
       }
-      yield Saga.put(Chat2Gen.createStartConversation({tlf: `/keybase/team/${teamname}`}))
-      if (state.chat2.pendingSelected) {
-        yield Saga.put(Chat2Gen.createExitSearch({canceled: true}))
-        yield Saga.put(Chat2Gen.createSetPendingMode({pendingMode: 'none'}))
-      }
+      yield Saga.put(Chat2Gen.createPreviewConversation({teamname}))
```

Comment:
still using preview (all extrernal calls should) but this is a special case where the pendign conversation never resolves and auto disappears (since we never start it) so we need to know to exit pending mode